README.md

```diff
@@ -52,29 +56,27 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ![CoatRack architecture overview](./spring-boot/admin/src/main/resources/static/images/coatrack-architecture-overview.png)
 
-## Install
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
 
-```console
-./build.sh
-```
-To run the CoatRack web application, use:
+## How to Build CoatRack Container Images from Source
+
+ The following prerequisites are required:
+
+* a Linux shell - Windows users can use WSL or the Git Bash shell instead
+* docker
+* docker-compose
+* OpenJDK 11
+* Maven 3.6.3 or higher
+
+
+
+There are two scripts in the `docker` directory helping you:
+
+* `build-and-push-images.sh` builds the docker image of each CoatRack modul from source and pushes the images to Dockerhub. This script shall only be used by the CI pipeline.
+* `build-and-deploy-images-locally.sh` builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
 
-```console
-./run-admin-application.sh
-```
-After starting up, the CoatRack web application will be accessible at `http://localhost:8080` 
 
-*Optional:* If you are a CoatRack developer and want to run the application using a custom Spring profile, you can pass the profile name to the run script as follows:
```

Comment:
This private profile information should not be deleted. Maybe it does not fit into this section, but in general we want to keep this information for developers. 

In doubt: create a follow-up issue as reminder to add this again in a later merge.

Reply:
In created the follow-up issue #195.